### PR TITLE
Mokhtar2019 branch

### PR DESCRIPTION
**Pull request by onoorzay**
Plz do a quick review before merging to make sure my changes don't break anything

### CODE REVIEW

File: AAC Events/AAC Events/AboutViewController.swift [hunk 1]
```diff
@@ -60,43 +61,128 @@ class AboutViewController: UIViewController, SideMenuItemContent {
...
 	}
 	
-	func getTheConferenceText() -> NSAttributedString {
+	func getOurCommittment() -> NSAttributedString {
 		let boldAttributes = [NSAttributedStringKey.font: boldFont]
 		let descriptionAttributes = [NSAttributedStringKey.font: descriptionFont]
         
-        let runningConfAttrString = NSMutableAttributedString()
+        let diveristyAttributes = [
+            NSAttributedStringKey.link: URL(string: "https://www.afghanamericanconference.org/diversity-1")!,
```
Comment (SiarNoorzay) [inline AAC Events/AAC Events/AboutViewController.swift:111]: These links are gonna be a problem. If they change them the app needs a new release to fix it. You should add them to dropbox and create a new endpoint that can just be a list of the URLS.

File: AAC Events/AAC Events/CommitteeViewController.swift [hunk 2]
```diff
@@ -31,47 +31,70 @@ class CommitteeViewController: UIViewController, SideMenuItemContent {
     
 
     
-    let orgNameArray: [String] = ["Zachia Nazarzai", "Salmon Hossein", "Yousuf Azam", "Susan Zaca", "Gina Karimi", "Homaira Hosseini", "Sara Noorzay", "Madina Salemi Nooristani", "Omar Aziz", "Sabrina Barkezai", "Samim Abedi", "Sophie Hossein", "Haroun Dada"]
-    let orgTitleArray: [String] = ["Executive Director", "Executive Director", "Finance Co-Chair, Development", "Communications Chair", "Applications Chair", "Community Initiatives Chair", "Community Engagement Chair", "Finance Co-Chair", "Programming Co-Chair", "Marketing Chair", "Professional Network Chair", "Programming Co-Chair", "Logistics Chair"]
+    let orgNameArray: [String] = ["Zachia Nazarzai", "Sara Noorzay", "Atiq Ahmadullah", "Gina Karimi", "Arzo Kaderi", "Mukhtar Mayan", "Nazieh Fazli", "Madina Salemi Nooristani", "Aresh Ramin", "Sabrina Noorzay", "Lida Azim", "Omar Aziz", "Sophie Hossein"]
```
Comment (SiarNoorzay) [inline AAC Events/AAC Events/CommitteeViewController.swift]: Wrong last name for Sabrina lol

File: AAC Events/AAC Events/Models/PeopleList.swift [hunk 3]
```diff
@@ -68,6 +70,12 @@ class Person: NSObject {
         } else {
             return nil
         }
+        //added
```
Comment (SiarNoorzay) [inline AAC Events/AAC Events/Models/PeopleList.swift]: Remove comments

File: AAC Events/AAC Events/PersonDetailsViewController.swift [hunk 4]
```diff
@@ -9,37 +9,38 @@
...
-    var sessionIDs: [String] = []
-    var personTitle: String! = "Title of the person"
-    var name: String! = "Name goes here"
-    var personDescription: String! = ""
-    var imageURL: String! = ""
-    var imageName: String! = ""
-    
+    //added
+    var person: Person!
+    //var sessionIDs: [String] = []
```
Comment (SiarNoorzay) [inline AAC Events/AAC Events/PersonDetailsViewController.swift]: remove comments

**Review by SiarNoorzay — CHANGES_REQUESTED** [review]
Its fine for the most part but remove commented code. It just makes things more messy and if its needed later its easy to get it from the history. 

Also don't add new lines randomly. It just makes things harder to read. Use them to separate chunks of related code. 

You can tell from the links that they aren't permanent and will definitely cause problems later.

File: AAC Events/AAC Events/AboutViewController.swift (hunk 1 above)
Comment (onoorzay) [reply AAC Events/AAC Events/AboutViewController.swift:111]: I think for this AAC its fine if the links are just in the app.

**Comment by onoorzay** [comment]
@Raiszard before you merge, can you check the about screen. There is an issue i am facing on the segmented control tabs. If you scroll halfway through the content on one tab and then click the next tab, the new tab starts halfway through the screen too. Couldn't figure it out.

**Pull request merged**